LICENSE

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why add this line？

Reply:
I juse refer to standard license , like dubbo ,spring and apache standard licenses.  we can keep consistency

Reply:
standard license without blank line in start

Reply:
i check many project  without blank line is ok. fixed